### PR TITLE
[Feature] Update initializer blueprint for ember-mocha 0.14

### PR DESCRIPTION
**Pull request by simonihmig**
Related to #16863

### CODE REVIEW

File: blueprints/initializer-test/mocha-rfc-232-files/__root__/__testType__/__path__/__name__-test.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+import { setupTest } from 'ember-mocha';
+import Application from '@ember/application';
+import { initialize } from '<%= modulePrefix %>/initializers/<%= dasherizedModuleName %>';
+<% if (destroyAppExists) { %>import destroyApp from '../../helpers/destroy-app';<% } else { %>import { run } from '@ember/runloop';<% } %>
+
+describe('<%= friendlyTestName %>', function() {
+  let hooks = setupTest();
```
Comment (Turbo87) [inline blueprints/initializer-test/mocha-rfc-232-files/__root__/__testType__/__path__/__name__-test.js]: do we need this? we don't seem to be using anything from the test context 🤔

Comment (Turbo87) [reply blueprints/initializer-test/mocha-rfc-232-files/__root__/__testType__/__path__/__name__-test.js]: hmm, looks like the QUnit variant is doing the same thing. @rwjblue do you remember why we use it there?

Comment (simonihmig) [reply blueprints/initializer-test/mocha-rfc-232-files/__root__/__testType__/__path__/__name__-test.js]: Yes, I made it do the same as the QUnit one for now, but also wondered about this. See my note here: https://github.com/emberjs/ember.js/pull/17412

Comment (simonihmig) [reply blueprints/initializer-test/mocha-rfc-232-files/__root__/__testType__/__path__/__name__-test.js]: @rwjblue do you? 👉 https://github.com/emberjs/ember.js/pull/17411#discussion_r244282578 
Would love to finish this and #17412, to update the TS blueprints as well! 😉

Comment (rwjblue) [reply blueprints/initializer-test/mocha-rfc-232-files/__root__/__testType__/__path__/__name__-test.js]: No, seems like we can/should remove from both mocha and qunit. Sorry for the delay here.

Comment (simonihmig) [reply blueprints/initializer-test/mocha-rfc-232-files/__root__/__testType__/__path__/__name__-test.js]: Removed `setupTest()`! Ready for final review.

Comment (simonihmig) [reply blueprints/initializer-test/mocha-rfc-232-files/__root__/__testType__/__path__/__name__-test.js]: Here for QUnit again: https://github.com/emberjs/ember.js/pull/17713

**Review by rwjblue — APPROVED** [review]

**Comment by rwjblue** [comment]
Hmm, looks like the blueprint tests are failing still.

**Comment by rwjblue** [comment]
Thank you @simonihmig!

**Pull request merged**